src/machine/mod.rs

```diff
@@ -1,34 +1,41 @@
 pub mod behavior;
 pub mod state;
 
+use self::state::mem::ByteAt;
 use self::state::State;
 
 pub struct Machine {
-    _state: State,
+    state: State,
     /* TODO add more fields... */
 }
 
 impl Machine {
     // TODO FYI: Machine uses the builder pattern.
 
     pub fn new() -> Self {
-        unimplemented!();
+        Machine {
+            state: State::new()
+        }
     }
 
-    pub fn load(self, _pos: u64, _data: &[u8]) -> Self {
+    pub fn load(mut self, pos: u64, data: &[u8]) -> Self {
```

Comment:
Maybe `dump` (from memory dump) is a better name than `data` since we load data and instructions.